docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
Did you have issues with that patch? I don't think Ship It needs any initialization, because there's no permissions to insert, nor necessary initial data.